Guidelines.md

```diff
@@ -1283,46 +1287,38 @@ There are three groups of changes that may happen to an API.
 
-A _breaking change_ is any change in the API that may cause client or service code making the API call to fail. Obvious examples of such a change are the removal of an endpoint, adding or removing a required field or changing the format of the body (from XML to JSON for example). Even though we recommend clients ignore new fields, there are many libraries and clients that fail when new fields are introduced. Removing an endpoint from an API is always a _breaking change_.  Adding a new endpoint is always an _evolutionary change_.  Changes to properties may be _evolutionary_ or _breaking_ depending on the type of change and whether the change is to an input parameter or output parameter:
+A _breaking change_ is any change in the API that may cause client or service code making the API call to fail. Obvious examples of such a change are the removal of an endpoint, adding or removing a required field or changing the format of the body (from XML to JSON for example). Removing an endpoint from an API is always a _breaking change_.  Adding a new endpoint is always an _evolutionary change_.  Changes to properties may be _evolutionary_ or _breaking_ depending on the type of change and whether the change is to an input parameter or output parameter:
 
 | Property change        | Input        | Output       |
 |:-----------------------|:------------:|:------------:|
 | Remove a property      | Breaking     | Breaking     |
-| Add optional property  | Evolutionary | Breaking     |
-| Add required property  | Breaking     | Breaking     |
+| Add optional property  | Evolutionary | Evolutionary |
+| Add required property  | Breaking     | Evolutionary |
 | Data type change       | Breaking     | Breaking     |
 | Format change          | Breaking     | Breaking     |
 | Integer widens         | Evolutionary | Breaking     |
 | Integer narrows        | Breaking     | Evolutionary |
 | Add new value to enum  | Evolutionary | Breaking     |
 | Remove value from enum | Breaking     | Breaking     |
-| Optional to required   | Breaking     | Breaking     |
+| Optional to required   | Breaking     | Evolutionary |
 | Required to optional   | Evolutionary | Breaking     |
 
-Breaking changes require prior approval of the Azure REST API review board and approval through the [Azure Global Breaking Change Policy][7]. In the case of deprecation, follow the [Azure Global Retirement Policy][7].  If the service is using SemVer for versioning, breaking changes constitute a major version change.
-
-Evolutionary changes do not require prior approval (but still need a version bump).  If the service is using SemVer for versioning, evolutionary changes constitute a minor version change.
+Breaking changes do not require any approvals, but *must* increment the API major version.
 
-#### 12.2.1 Why Centrify recommends conservative API versioning
+Evolutionary changes do not require any approvals or version changes.
```

Comment:
Evolutionary changes do not require any approvals or version changes, provided change is 100% backward compatible for all use cases.